app/helpers/homepage_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomepageHelper
```

Comment:
let's not add files that are not being used

Reply:
done